README.md

```diff
@@ -40,6 +40,14 @@ To run the test-suite locally, first complete the bootstrap step as outline abov
 tox -e tests
 ```
 
+### Documentation
+
+To generate html documentation locally in docs/_build/html, run the following command
+```shell 
```

Comment:
minor: I would add an empty line between the text and the code block

```
To generate…

```shell
…
```

Reply:
done